src/main/resources/db/migration/V013__Add_new_fields.sql

```diff
@@ -0,0 +1,18 @@
+ALTER TABLE envelopes
+  ADD COLUMN caseNumber VARCHAR(100) NULL;
+
+ALTER TABLE non_scannable_items
+  ADD COLUMN documentControlNumber VARCHAR(100) NOT NULL;
+
+ALTER TABLE payments
+  ADD COLUMN paymentInstrumentNumber VARCHAR(100) NULL,
+  ADD COLUMN sortCode VARCHAR(10) NULL,
+  ADD COLUMN accountNumber VARCHAR(50) NULL;
+
+ALTER TABLE scannable_items
+  ALTER COLUMN documentControlNumber SET NOT NULL;
+
+UPDATE payments
+  SET sortCode = '112233',
+      accountNumber = '12345678'
```

Comment:
comment explaining why would be nice.
Also, if you set those value, should this column be `NOT NULL`?

Reply:
@rkondratowicz The mandatory fields are different for each payment method. sortcode and account number fields are mandatory only for 'Cheque' payment. So the above update statement is to provide default values for existing Cheque payments data.